deployment/molecule/ultimate-native-to-erc/converge.yml

```diff
@@ -0,0 +1,3 @@
+---
+- import_playbook: ../../site.yml
+- import_playbook: ./overwrite-docker-compose.yml
```

Comment:
The modification of Oracle's `docker-compose.yml` file is necessary in order to add `ultimate` network - joining the containers with parity containers